### ISSUE TITLE
Enhancement of KitodoScript: copyData

### ISSUE THREAD

**Issue by andre-hohmann** [kitodo/kitodo-production#2239]
# Description

## 1 Addition of metadata and values
The KitodoScript “copyData” can be used to add values to metadata elements in several Kitodo-processes [TS-1257_BA_Erweiterung-Spielzeiten_D.pdf](https://github.com/kitodo/kitodo-production/wiki/Zeutschel_Doku/TS-1257_BA_Erweiterung-Spielzeiten_D.pdf), page 3. Generally, it works to add values to metadata on topStruct or firstChild elements. 
However, it works only if values do not contain blanks as for example in “CC BY-SA 4.0”. If the value is “CCBY-SA4.0”, it works. 

## 2 Copy of several metadata elements
In addition, the KitodoScript “copyData” can be used to copy metadata elements from one structure element to another. 
However, if there are several metadata elements of the same type (for example: “digital collection”), it is only possible to copy one of the several metadata elements to another structure element. 

# Examples 
## Addition of metadata and values
```
action:copyData
@LegalNoteAndTermsOfUse = " CCBY-SA4.0";
```
-> It does work 

```
action:copyData
@LegalNoteAndTermsOfUse = " CC BY-SA 4.0";
```
-> It does not work

# Request
For a better general usability, the copyData-script should be enabled to 
- process values containing blanks
- copy several metadata elements of the same type from one structure element to another structure element

# Additional information
[TS-1257_BA_Erweiterung-Spielzeiten_D.pdf](https://github.com/kitodo/kitodo-production/wiki/Zeutschel_Doku/TS-1257_BA_Erweiterung-Spielzeiten_D.pdf)

**Comment by matthias-ronge** [comment]
This has been fixed in version 3 with #3324, the patch could be ported back.

**Comment by andre-hohmann** [comment]
As in Kitodo.Production 3 other [scripts](https://github.com/kitodo/kitodo-production/wiki/Metadaten-Skripte) exists to add, delete, ... metadata, this issue is closed as not planned.
